Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/SubmissionRepository.kt

```diff
@@ -67,4 +69,12 @@ object SubmissionRepository {
     fun setTeletan(teletan: String) {
         LocalData.teletan(teletan)
     }
+
+    fun setVisitedCountries(countries: List<String>) {
+        visitedCountries.postValue(countries)
+    }
+
+    fun setConsentToFederation(consent: Boolean) {
+        consentToFederation.postValue(consent)
+    }
```

Comment:
Just to double-check: the keyword "persistence" in the commit description does not refer to storing it in LocalData etc, correct?

Reply:
yeah, thats a wording mistake, as of right now the spec contains no definition for additional local storage properties